cmd/ctr/shim.go

```diff
@@ -133,11 +137,15 @@ var shimStartCommand = cli.Command{
 	Name:  "start",
 	Usage: "start a container with a shim",
 	Action: func(context *cli.Context) error {
-		service, err := getShimService()
+		var (
```

Comment:
well, now with the `service, err =` inside the var blocks it does not match any of our other code.  we never do this anywhere else.  Could you change that back but keep the context as a var declared at the top so that we use the same context for all RPCs in the commands?